test/e2e/testdata/sigstore-key.key

```diff
@@ -0,0 +1,11 @@
+-----BEGIN ENCRYPTED COSIGN PRIVATE KEY-----
```

Comment:
Do we really want to commit private keys for testing? IMO keys should be automatically generated before the tests and we should not used fixed ones.

Reply:
The format for this is `cosign`-specific. Right now, to generate this, we would need to install `cosign` during tests.

Eventually Skopeo will be taught to generate the key pair, and/or we might accept keys in more standardized formats, but that doesn’t exist yet.

(To be explicit, this key pair was generated specifically for the tests and has no other use.)

Reply:
I am fine with this, looks like we already have other private keys committed anyway.

My main concern is that we are testing this *one* key and not random generated keys like normal users would do.